### ISSUE TITLE
Comments following deployment on dev

### ISSUE THREAD

**Issue by lpalbou** [geneontology/noctua-annotation-review#36]
First, overall looks great and glad to see that finally on dev.

## Things to fix / discuss
1. When there are no activities, like in this model, show a message stating so, otherwise it could feel like there was some errors while loading the model

![Screen Shot 2021-04-05 at 9 22 02 AM](https://user-images.githubusercontent.com/24249870/113597318-638ff300-95f0-11eb-8d8a-eed67203b5f1.png)

2. the height of that grey bar "ACTIVITIES" is really too big, I would decrease to 30px; still feel a bit clunky but not critical

3. I am surprised as I thought the "ACTIVITIES" bar was to list activities... and yet I find a "BP Annotation" and I imagine CC Annotation can appear there too. Seems like either that grey bar is useless OR you want to have multiple grey bars, one for activity, one for bp annotation and one for cc annotation (to show only when you have such annotations). And if it's the logic, then you don't need to show these badges "Activity Unit", "BP Annotation", "CC Annotation" as they would already be grouped under a same banner.
![Screen Shot 2021-04-05 at 9 24 04 AM](https://user-images.githubusercontent.com/24249870/113597524-ad78d900-95f0-11eb-9c80-b6984d260a6a.png)

4. I was in review, clear my basket and.. Then decided on the search icon (top left) and it showed me that:
![Screen Shot 2021-04-05 at 9 37 22 AM](https://user-images.githubusercontent.com/24249870/113598798-87eccf00-95f2-11eb-9485-c97b9b9c8f5e.png)

I believe clicking on the search icon should switch you back to the search page, otherwise it's confusing.

The culprit is still that bar :
![Screen Shot 2021-04-05 at 9 38 57 AM](https://user-images.githubusercontent.com/24249870/113598962-c08ca880-95f2-11eb-968d-af77392c7b01.png)

I believe we would have to redesign that later, as both this bar and the left bar control what you should see at the center.

5. in the same aspect, I don't think it's intuitive/clear that after selecting models, you should click on the review button on the top:
![Screen Shot 2021-04-05 at 9 40 26 AM](https://user-images.githubusercontent.com/24249870/113599078-f598fb00-95f2-11eb-97c9-e43c11571b93.png)

Not for this V1, but I would suggest that whenever models are selected, a floating colorful button on the bottom right appears and state something like: "X models selected - Click here to review them". The fact the button appears upon model selection would help people know where to click for the next step right away

6. I mentioned above I cleared my basket ("Clear List") in which I modified lipid transport into ion transport. But even though the modal state that any changes would not be saved... When I reselect those models, I still see my modifications:
![Screen Shot 2021-04-05 at 9 43 40 AM](https://user-images.githubusercontent.com/24249870/113599395-693b0800-95f3-11eb-8deb-9a1a4120eeda.png)

So something is wrong here, maybe you don't call reset on the models ?

And indeed, even after a refresh of the page, you do see there are pending changes:
![Screen Shot 2021-04-05 at 9 47 46 AM](https://user-images.githubusercontent.com/24249870/113599803-fd0cd400-95f3-11eb-8512-357277f3a58b.png)

Note that clicking on  "Undo changes" in the list of models to the left does reset the model changes. 
![Screen Shot 2021-04-05 at 9 48 28 AM](https://user-images.githubusercontent.com/24249870/113599982-39d8cb00-95f4-11eb-8dd5-f3ab23121eb8.png)

7.  we don't want to have both "List" (bottom left) and "Basket" (title of modal). I prefer "List" as we are not an ecommerce 
![Screen Shot 2021-04-05 at 9 44 49 AM](https://user-images.githubusercontent.com/24249870/113599519-92f42f00-95f3-11eb-821f-189f9ea6beec.png)

**Comment by tmushayahama** [comment]
For Number 3)
Separating the Activities into group is not idea because the GPs are sorted alphanumerical. And its easy to visualize your GPs. @vanaukenk @vanaukenk can comment more on this.

The "activities bar" is there for the menu on the right side and clear separation between model list and model's contents list. I can decrease height to 30px.

**Comment by tmushayahama** [comment]
As for number 6) The reasoning was if you clear list nothing will be changed, it's like unselecting model. Neither "undo" or "save" will happen. It will just remove it from the basket untouched. The idea is because if you select someone's already edited model, we don't want to force save or undo just to unselect it

**Comment by tmushayahama** [comment]
I agree with the other issues especially 4 and 5, the workflow is not too smooth. We can discuss it soon if its only UI and quick fix for V1?

**Comment by tmushayahama** [comment]
Also we have to revisit the popups wording so they are all clear to what will happen (number 6)

**Comment by lpalbou** [comment]
For 6: 
<img width="486" alt="Screen Shot 2021-04-05 at 12 49 48 PM" src="https://user-images.githubusercontent.com/24249870/113619105-69e09800-960d-11eb-8a7e-547ac494c5dc.png">

So the second sentence is wrong; all your changes are actually saved but not stored. So nothing that was done was lost or reset, which may be an issue. Probable resolution: change the text, possibly change the actions too, eg: clear list without reset; clear list and reset; cancel.  Need @ukemi & @vanaukenk feedback on what they want exactly here.

For 3:
<img width="1077" alt="Screen Shot 2021-04-05 at 12 55 44 PM" src="https://user-images.githubusercontent.com/24249870/113619794-45d18680-960e-11eb-902b-99d5013c9c41.png">

That's an awful lot of lost space just to fold/unfold models. Either we use that grey bar to fold activities together (like the title suggested) and have equivalent for BP & CC, or this should go away. If we remove the bar and still want the ability to fold/unfold everything, it's usually done by shift clicking the arrow up/down to fold/unfold and it's easy to teach to curators and is anyhow not blocking in their workflow.

**Comment by tmushayahama** [comment]
We will need more discussion on this. I think that bar should not be removed. 
1) It is a divider to know that one is inside the a model or list of models as the model title and activity titles looks the same
2) The menu on the right contains more things that just the 2 options and I don't think keyboard and mouse shortcut binding is a way to go.
3) Most importantly, it is also a status bar like loading messages and errors like below

![image](https://user-images.githubusercontent.com/2273929/113646002-c8713a80-963c-11eb-8e49-68669092467c.png)
 
or saving model etc
![image](https://user-images.githubusercontent.com/2273929/113646521-d5daf480-963d-11eb-9d7f-5dc918101f56.png)



And mostly, I think we have lot of vertical space, and so not worried about letting it there.
